plugins/dashboard/explorer_routes.go

```diff
@@ -142,8 +152,60 @@ func findMessage(messageID message.Id) (explorerMsg *ExplorerMessage, err error)
 	return
 }
 
-func findAddress(address string) (*ExplorerAddress, error) {
-	return nil, fmt.Errorf("%w: address %s", ErrNotFound, address)
+func findAddress(strAddress string) (*ExplorerAddress, error) {
+
+	address, err := address.FromBase58(strAddress)
+	if err != nil {
+		return nil, fmt.Errorf("%w: address %s", ErrNotFound, strAddress)
+	}
+
+	outputids := make([]ExplorerOutput, 0)
+	// get outputids by address
+	for id, cachedOutput := range valuetransfers.Tangle().OutputsOnAddress(address) {
```

Comment:
Consider using `valuetransfers.Tangle().OutputsOnAddress(<bla>).Consume()` so you don't have to do `defer`s inside a for loop.